Dockerfile

```diff
@@ -1,12 +1,15 @@
 FROM registry.redhat.io/ubi8/ubi-minimal
 
+ARG PYTHON_PIP_VERSION=23.3
+
 RUN microdnf install \
     --disableplugin=subscription-manager \
     --nodocs -y python311 tar gzip gcc python3.11-devel libpq-devel
 
 # Install poetry in separate virtual env
 ENV POETRY_HOME=/opt/poetry
 RUN python -m venv $POETRY_HOME &&\
+    $POETRY_HOME/bin/pip install --upgrade pip==$PYTHON_PIP_VERSION setuptools wheel && \
```

Comment:
Do we require `POETRY_HOME`?

Reply:
Yes, as I am trying to upgrade these packages only within the boundary of given venv.